dynamic/run.sh

```diff
@@ -118,5 +118,21 @@ if [ -d /host/proc ]; then
   export INSTANA_AGENT_PROC_PATH=/host/proc
 fi
 
+if [ -f /root/crashReport.sh ]; then
+  cp /root/crashReport.sh /opt/instana/agent/crashReport.sh
+
+  # Rewrite the Karaf script to make sure spaces in JAVA_OPTS are properly escaped (JAVA_OPTS is not within quotes)
+  sed -i "s|\ \${JAVA_OPTS}\ |\ \"\${JAVA_OPTS}\"\ |g" /opt/instana/agent/bin/karaf
```

Comment:
Karaf script does not wrap the `JAVA_OPTS` in quotes, I've got a pending PR for that: https://github.com/apache/karaf/pull/955
Without having it wrapped in quotes, its basically impossible to supply the parameter `-XX:OnError="crashReport.sh %p"` because of the space in the argument.

The `sed` command will replace the two occurrences and wrap them.